docs/pair-mob-programming.md

```diff
@@ -0,0 +1,109 @@
+---
+id: pair-mob-programming
+title: Desarrollo de Software en Parejas o Grupos
```

Comment:
@steph222 

Este titulo esta muy largo , se pasa a dos lineas , podes ver el preview aqui:
https://deploy-preview-216--guias.netlify.app/docs/pair-mob-programming

lo mismo sucede en la version en ingles:
https://guide.eoscostarica.io/docs/pair-mob-programming

Sugiero cambiar ambos títulos a :

`Programación  en Parejas o Grupos`
y
`Pair and Mob Programming`

Reply:
Claro no hay problema!!